test/SerialIOTest.cpp

```diff
@@ -815,7 +815,11 @@ TEST_CASE( "hdf5_dtype_test", "[serial][hdf5]" )
         s.setAttribute("vecUint64", std::vector< uint64_t >({18446744073709551614u, 18446744073709551615u}));
         s.setAttribute("vecFloat", std::vector< float >({0.f, 3.40282e+38f}));
         s.setAttribute("vecDouble", std::vector< double >({0., 1.79769e+308}));
+#ifdef _MSC_VER
+        s.setAttribute("vecLongdouble", std::vector< long double >({0.L, 1.49769e+308L}));
```

Comment:
This could be replaced nicely with [```std::numeric_limits::max```](http://en.cppreference.com/w/cpp/types/numeric_limits/max). This would make it compatible across all compilers (_that conform to the C++ standard_).

Reply:
Yep, just a quick hack so far :)